converters/fromZigbee.js

```diff
@@ -3409,6 +3409,17 @@ const converters = {
             return {moving: msg.data['acceleration'] === 1 ? true : false};
         },
     },
+    smartthings_accelerometer: {
```

Comment:
Given it's processing the same cluster, would it make more sense for this code to go into `smartthings_acceleration` instead?

Reply:
It might. Since I'm not vary familiar with zigbee internals or the conventions of this repo, I separated it out. Could use a recommendation on this.

Reply:
done